tests/system/common.py

```diff
@@ -790,3 +790,26 @@ def task_by_name(tasks, name):
 
 async def assert_event(event_type, event_stream, within=10):
     await asyncio.wait_for(find_event(event_type, event_stream), within)
+
+
+def kill_process_on_host(hostname, pattern):
```

Comment:
Should we use this method in other places as well?

Reply:
No, ideally not at all or as little as possible. `kill_task(host, '[h]ttp.server')` will kill *all http.server`s on the host - that makes running tests in parallel impossible.

Reply:
Yes. But one can define the pattern. E.g. how does it related to `shakedown.kill_process_on_host(host, '[s]leep 1000')` in `marathon_common_tests.py`?